getting_started.ipynb

```diff
@@ -0,0 +1,840 @@
+    "To smear our original spectrum and create the new `Spectra` object `smeared_spectrum`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "smeared_spectrum = smearer.weighted_smear(spectrum_1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "this should hopefully only create a couple of seconds."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following code shows you how to make a simple script, using matplotlib, to overlay the original and smeared spectra."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "\n",
+    "def overlay_spectra(original, smeared, dimension=\"energy_mc\", fignum=1):\n",
+    "    \"\"\" Overlay original and smeared spectra.\n",
+    "    \n",
+    "    Args:\n",
+    "      original (echidna.core.spectra.Spectra): Original spectrum.\n",
+    "      smeared (echidna.core.spectra.Spectra): Smeared spectrum.\n",
+    "      dimension (string, optional): Dimension to project onto.\n",
+    "        Default is \"energy_mc\".\n",
+    "      fignum (int, optional): Figure number, if producing multiple\n",
+    "        figures. Default is 1.\n",
+    "\n",
+    "    Returns:\n",
+    "      matplotlib.figure.Figure: Figure showing overlaid spectra.\n",
+    "    \"\"\"\n",
+    "    fig = plt.figure(num=fignum)\n",
+    "    ax = fig.add_subplot(1, 1, 1)\n",
+    "    \n",
+    "    par = original.get_config().get_par(dimension)\n",
+    "    width = par.get_width()\n",
+    "\n",
+    "    # Define array of bin boundaries (1 more than number of bins)\n",
+    "    bins = np.linspace(par._low, par._high, par._bins+1)\n",
+    "    # Define array of bin centres\n",
+    "    x = bins[:-1] + 0.5*width\n",
+    "\n",
+    "    # Overlay two spectra using projection as weight\n",
+    "    ax.hist(x, bins, weights=original.project(dimension), histtype=\"stepfilled\",\n",
+    "            color=\"RoyalBlue\", alpha=0.5, label=original._name)\n",
+    "    ax.hist(x, bins, weights=smeared.project(dimension), histtype=\"stepfilled\",\n",
+    "            color=\"Red\", alpha=0.5, label=smeared._name)\n",
+    "\n",
+    "    # Add label/style\n",
+    "    plt.legend(loc=\"upper right\")\n",
+    "    plt.ylim(ymin=0.0)\n",
+    "    plt.xlabel(dimension + \" [\" + par.get_unit() + \"]\")\n",
+    "    plt.ylabel(\"Events per \" + str(width) + \" \" + par.get_unit() + \" bin\")\n",
+    "    return fig"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fig1 = overlay_spectra(spectrum_1, smeared_spectrum, fignum=4)\n",
+    "plt.show()"
```

Comment:
The plot didn't appear. This was written:
ValueError                                Traceback (most recent call last)
<ipython-input-14-3378db744f5a> in <module>()
----> 1plot.plot_surface(spectrum_1, "energy_mc", "radial_mc")

/.data/snoplus/evelina/virtual_echidna/echidna/echidna/output/plot.pyc in plot_surface(spectra, dimension1, dimension2)
    192     """
    193     fig = plt.figure()
--> 194axis = fig.add_subplot(111, projection='3d')
    195     index1 = spectra.get_config().get_index(dimension1)
    196     index2 = spectra.get_config().get_index(dimension2)

/.data/snoplus/evelina/virtual_echidna/lib/python2.7/site-packages/matplotlib/figure.pyc in add_subplot(self, *args, **kwargs)
    944         else:
    945             projection_class, kwargs, key = process_projection_requirements(
--> 946                 self, *args, **kwargs)
    947 
    948             # try to find the axes with this key in the stack

/.data/snoplus/evelina/virtual_echidna/lib/python2.7/site-packages/matplotlib/projections/__init__.pyc in process_projection_requirements(figure, *args, **kwargs)
     96 
     97     if isinstance(projection, six.string_types) or projection is None:
---> 98projection_class = get_projection_class(projection)
     99     elif hasattr(projection, '_as_mpl_axes'):
    100         projection_class, extra_kwargs = projection._as_mpl_axes()

/.data/snoplus/evelina/virtual_echidna/lib/python2.7/site-packages/matplotlib/projections/__init__.pyc in get_projection_class(projection)
     65         return projection_registry.get_projection_class(projection)
     66     except KeyError:
---> 67         raise ValueError("Unknown projection '%s'" % projection)
     68 
     69 

ValueError: Unknown projection '3d'